service/src/main/java/bio/terra/workspace/service/resource/referenced/ReferencedResourceService.java

```diff
@@ -248,4 +268,32 @@ public ReferencedResource cloneReferencedResource(
     return createReferenceResourceForClone(
         destinationResource, sourceReferencedResource, userRequest);
   }
+
+  private void validateDestinationWorkspacePolicies(
+      UUID sourceWorkspaceId, UUID destinationWorkspaceId, CloudPlatform platform) {
+    TpsPaoUpdateResult dryRunResults =
+        tpsApiDispatch.mergePao(sourceWorkspaceId, destinationWorkspaceId, TpsUpdateMode.DRY_RUN);
+
+    if (!dryRunResults.getConflicts().isEmpty()) {
+      throw new ConflictException("Policy merge has conflicts");
+    }
+
+    if (platform == CloudPlatform.ANY) {
+      logger.warn("Cannot validate regions for CloudPlatform.ANY");
```

Comment:
I think this is a normal case for things like datarepo snapshots and github repos. So maybe not worth logging.